roachpb/data.go

```diff
@@ -554,6 +554,68 @@ func (v Value) computeChecksum(key []byte) uint32 {
 	return sum
 }
 
+// PrettyPrint returns the value in a human readable format.
+// e.g. `Put /Table/51/1/1/0 -> /TUPLE/2:Int/2/1:Int/3`
```

Comment:
Can you explain the value format here? I can only guess what `2:Int/2/1:Int/3` is.